### ISSUE TITLE
Voice message sharing works bad

### ISSUE THREAD

**Issue by 532910** [vector-im/element-android#6254]
### Steps to reproduce

1. Record an audio message.
2. Share it.

![image](https://user-images.githubusercontent.com/14617699/172505506-895e7f5e-cec2-40d3-972d-d2cff726382c.png)

### Operating system version

Lineage+microG 18.1 on Android 11

### Application version and app store

Element 1.4.16

### Will you send logs?

No

### Are you willing to provide a PR?

No

**Comment by ouchadam** [comment]
will need some more information to help diagnose the problem, from the screenshot it looks like an audio file has been uploaded rather than a voice message 


| VOICE MESSAGE | AUDIO FILE |
| --- | --- |
![2022-06-14T16:40:01,418497524+01:00](https://user-images.githubusercontent.com/1848238/173619178-a94d8c2a-9b37-4423-850b-cfee0bf46f95.png)|![2022-06-14T16:40:48,062198662+01:00](https://user-images.githubusercontent.com/1848238/173619171-e5680992-cf50-4a4b-9da5-3b199f4ebc2a.png)

**Comment by 532910** [comment]
> from the screenshot it looks like an audio file has been uploaded rather than a voice message

Yes, this issue exactly about this. A shared voice message looks like an audio file.

What more information do you need?

**Comment by ouchadam** [comment]
as there isn't much information in the ticket it's quite easy to misinterpret the problem, more detail about the reproduction steps would be helpful, does the issue happen every time for example?

it's hard to know from the description and image if an external voice recording has been shared via Element or a voice message using the built in functionality is rendering as an audio file

**Comment by thewompt** [comment]
Issues I am having:

* Extreme delay in playing messages, sometimes taking 60s+ to start playback.
* Incorrect durations
* No automatic play through in window, must click each element on the play button individually
* Play button is a small target, and clicking elsewhere in the element does different things.
* Occassionaly app will stop trying to play messages and requires a restart to continue conversation.

That's all I can think of at the moment.

Voice messaging bad.

**Comment by magrimuc** [comment]
Because of 2 minutes length limit I record voice via system build in voice recorder and upload the file. This is not the most glamorous handling for element...
